input.css

```diff
@@ -54,15 +54,16 @@ body {
   
 pre {
   padding: 1rem !important;
+  background-color: rgb(39, 40, 34) !important;
```

Comment:
Hugo uses [Chroma](https://gohugo.io/content-management/syntax-highlighting/) to apply syntax highlights and styles for code snippets. If a language is not supported, no style is applied. Specifying the bg color ensures that all code snippets have the same bg color.